src/overlays/actors/ovl_Obj_Warp2block/z_obj_warp2block.c

```diff
@@ -43,36 +43,280 @@ const ActorInit Obj_Warp2block_InitVars = {
     (ActorFunc)ObjWarp2block_Update,
     (ActorFunc)ObjWarp2block_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Warp2block/func_80BA1DF0.s")
+typedef struct {
+    /* 0x00 */ f32 scale;
+    /* 0x04 */ f32 focus;
+    /* 0x08 */ s16 params;
+} Warp2BlockSpawnData; // size = 0x0A
```

Comment:
This struct actually ends up with a size of 0x0C bytes because of padding
```suggestion
} Warp2BlockSpawnData; // size = 0x0C
```